merge.go

```diff
@@ -217,11 +217,12 @@ func (db *DB) mergeWorker() {
 
 func (db *DB) isPendingMergeEntry(entry *Entry) bool {
 	if entry.Meta.Ds == DataStructureTree {
-		bptIdx, exist := db.BTreeIdx[string(entry.Bucket)]
+		idx, exist := db.BTreeIdx[string(entry.Bucket)]
```

Comment:
Why just change the name of this variable here?

Reply:
Because it is not a b plus tree idx now🤣

Reply:
i see.